.vscode/tasks.json

```diff
@@ -5,7 +5,7 @@
 			"label": "Serve DDC",
 			"type": "shell",
 			"options": {
-				"cwd": "${workspaceFolder}/packages/devtools",
+				"cwd": "${workspaceFolder}/packages_app/devtools",
 			},
 			"command": "pub",
```

Comment:
Should this be devtools_app?

Reply:
yes. package:devtools is now just a placeholder for the deployed artifacts. When debugging, you just need package:devtools_app.

Reply:
I mean should this say `packages/devtools_app` instead of `packages_app/devtools`

Reply:
done... wow.... `packages_app`.